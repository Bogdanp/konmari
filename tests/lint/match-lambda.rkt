#lang racket/base

(require racket/match)

(define foo
  (match-lambda
    [#t
     (define a 1)
     (+ a 2)]))

(define bar
  (match-lambda
    [#t
     (define a 2)
     (+ a 1)]))
