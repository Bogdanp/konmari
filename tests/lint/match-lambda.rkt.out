/match-lambda.rkt:5:9:warning:identifier 'foo' is never used
/match-lambda.rkt:11:9:warning:identifier 'bar' is never used
